homework/tests/example.spec.js

```diff
@@ -15,17 +21,176 @@ const data = [
 ]
 
 
+//Verify if adding works properly using whole numbers
+data.forEach(version => {
+  test.describe(version + ': ADD', () => {
+    test('Adding 1 and 2 result in 3', async ({ page }) => {
+      await page.goto(Calculatorpage);
```

Comment:
Since this is a common step throughout all test cases - you could use `beforeEach` hook to navigate to this URL